sssom/parsers.py

```diff
@@ -140,6 +141,28 @@ def read_obographs_json(
     )
 
 
+def read_snomed_complex_map_tsv(
+    file_path: str,
+    prefix_map: Dict[str, str] = None,
+    meta: Dict[str, str] = None,
+    filter_by_confident_mappings=True
```

Comment:
Based on your request to filter only mapping rows that contain `'ALWAYS <code>'`, and possible future flexibility to fine tune the mappings, I added a param here and just set it to `True`.

Reply:
@matentzn